### DISCUSSION TITLE
I am unable to setup "typescript-is" with nextjs.

### DISCUSSION THREAD

**Discussion by Morritz** [vercel/next.js#24996] [Help]
https://www.npmjs.com/package/typescript-is
I tried many configurations, but so far i did not get any successful run.
```
const typescriptIsTransformer = require('typescript-is/lib/transform-inline/transformer').default

module.exports = {
  future: {
    webpack5: true,
  },
    webpack: (config, options) => {
      config.module.rules.push({
        test: /\.tsx?$/,
        exclude: /node_modules/,
        loader: 'ts-loader',
        options: {
            getCustomTransformers: program => ({
                before: [typescriptIsTransformer(program)]
            })
        }
    })
  
      return config
    },
  }

```
  I use following next.config.js , but it is not sufficient. I assume that it might be hard to customize as nextjs may use some predfined build process, so I would like her to ask for help with this subject someone that either develops this framework or is really experienced with it.

**Comment by piotaixr** [comment]
I would be interested to know how to do it (if it is possible to do so).

From what I understand, Next.js 11 is using SWC as typescript loader for faster builds making the use of typescript-is impossible without replacing SWC.
